<!--
  Copyright JS Foundation and other contributors, http://js.foundation

  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at

  http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
-->

<script type="text/html" data-help-name="catch">
    <p> Fängt Fehler von Nodes auf derselben Registerkarte ab. </p>
    <h3> Ausgaben </h3>
    <dl class="message-properties">
        <dt> error.message <span class="property-type"> Zeichenfolge </span> </dt>
        <dd> die Fehlernachricht. </dd>
        <dt> error.source.id  <span class="property-type"> Zeichenfolge </span> </dt>
        <dd> Die ID des Nodes, der den Fehler ausgelöst hat. </dd>
        <dt> error.source.type  <span class="property-type"> Zeichenfolge </span> </dt>
        <dd> Der Typ des Nodes, der den Fehler ausgelöst hat. </dd>
        <dt> error.source.name  <span class="property-type"> Zeichenfolge </span> </dt>
        <dd> Der Name des Nodes (falls festgelegt), der den Fehler ausgelöst hat. </dd>
    </dl>
    <h3> Details </h3>
    <p> Wenn ein Node bei der Verarbeitung einer Nachricht einen Fehler auslöst, wird der Flow in der Regel
       angehalten. Dieser Node kann verwendet werden, um diese Fehler abzufangen und sie mit einem
       dedizierten Flow zu bearbeiten. </p>
    <p> Der Node fängt standardmäßig Fehler ab, die von einem beliebigen Node auf derselben Registerkarte ausgelöst werden. Alternativ
   kann er an einen bestimmten Node gebunden werden. </p>
    <p> Wenn ein Fehler ausgelöst wird, empfangen alle übereinstimmenden Fehlerabfang-Node die Nachricht. </p>
    <p> Wenn ein Fehler in einem Subflow ausgelöst wird, wird der Fehler von einem Fehlerabfang-Node
       innerhalb des Subflows abgefangen. Wenn keine vorhanden sind, wird zu dem Tab geleitet, 
       in der sich die Subflow-Instanz befindet. </p>
   <p> Wenn die Nachricht bereits über eine Eigenschaft  <code>error</code>  verfügt, wird sie nach  <code>_error</code> kopiert. </p>
</script>
